pyro/poutine/handlers.py

```diff
@@ -306,14 +306,16 @@ def escape(  # type: ignore[empty-body]
 def equalize(
     sites: Union[str, List[str]],
     type: Optional[str],
-) -> ConditionMessenger: ...
+    keep_dist: Optional[bool],
```

Comment:
I think `keep_dist` should have just a `bool` type, right? It should have the same type as in the `EqualizeMessenger` class definition.

Reply:
You are right they should have the same type, so I changed the type in the `EqualizeMessenger` class definition to `Optional[bool]`, as `keep_dist` is optional.